packages/react-devtools-shared/src/devtools/views/Profiler/RecordToggle.js

```diff
@@ -30,13 +30,17 @@ export default function RecordToggle({disabled}: Props): React.Node {
     className = styles.ActiveRecordToggle;
   }
 
+  const isMac = navigator.platform.toUpperCase().indexOf('MAC') >= 0;
```

Comment:
React DevTools frontend could be launched also outside of the browser environment. For example, inside Electron wrapper.

Does Electron support `navigator` global? You can launch React DevTools in Electron with `yarn start` in `react-devtools` package.